content/source/docs/cloud/integrations/service-now/service-catalog.html.md

```diff
@@ -0,0 +1,160 @@
+
+To make infrastructure available to your users, you must add one or more VCS
+repositories containing Terraform configuration to the Terraform service
+catalog.
+
+1. In ServiceNow, open the "Terraform > VCS Repositories" table by searching for
+   "terraform" in the left-hand navigation.
+1. Click "New" to add a VCS repository.
+    - Name: The name for this repository. This name will be visible to end
+      users, and does not have to be the same as the repository name as defined
+      by your VCS provider. Ideally it will succinctly describe the
+      infrastructure that will be provisioned by Terraform from the repository.
+    - OAuth Token ID: The OAuth Token ID that from your Terraform Cloud
+      organization's VCS providers settings. This ID specifies which VCS
+      provider configured in Terraform Cloud hosts the desired repository.
+    - Identifier: The VCS repository that contains the Terraform configuration
+      for this workspace template. Repository identifiers are determined by your
+      VCS provider; they typically use a format like
+      `<ORGANIZATION>/<REPO_NAME>` or `<PROJECT KEY>/<REPO_NAME>`. Azure DevOps
+      repositories use the format `<ORGANIZATION>/<PROJECT>/_git/<REPO_NAME>`.
+    - The remaining fields are optional.
+      - Branch: The branch within the repository, if different from the default
+        branch.
+      - Working Directory: The directory within the repository containing
+        Terraform configuration.
+      - Terraform Version: The version of Terraform to use. This will default to
+        the latest version of Terraform supported by your Terraform Cloud
+        instance.
+1. Click "Submit".
+
+![Screenshot: ServiceNow New VCS Repository](./images/service-now-vcs-repository.png "Screenshot of the ServiceNow Terraform New VCS Repository page")
+
+## Configure a Variable Set
+
+Most Terraform configuration can be customized with Terraform variables or
+environment variables. You can create a Variable Set within ServiceNow to
+contain the variables needed for a set of configuration. Your Terraform Admin
+should provide these to you.
+
+1. In ServiceNow, open the "Service Catalog > Variable Sets" table by searching for
+   "variable sets" in the left-hand navigation.
+1. Click "New" to add a Variable Set.
+1. Select "Single-Row Variable Set".
+    - Title: User-visible title for the variable set.
+    - Internal name: The internal name for the variable set.
+    - Order: The order in which the variable set will be displayed.
+    - Type: Should be set to "Single Row"
+    - Application: Should be set to "Terraform"
+    - Display title: Whether the title is displayed to the end user.
+    - Layout: How the variables in the set will be displayed on the screen.
+    - Description: A long description of the variable set.
+1. Click "Submit" to create the variable set.
+1. Find and click on the Title of the new variable set in the Variable Sets
```

Comment:
arguably nocaps on "title", but I'll accept whatever your best judgement is on these servicenow objects.